internals/generators/container/index.js

```diff
@@ -44,6 +44,11 @@ module.exports = {
     name: 'wantMessages',
     default: true,
     message: 'Do you want i18n messages (i.e. will this component use text)?',
+  }, {
+    type: 'confirm',
+    name: 'wantLoadable',
+    default: true,
+    message: 'Do you want an async loader?',
```

Comment:
Perhaps we should add something like `If you have enabled reducer or sagas, you most likely do.`?